config/src/main/java/io/scalecube/config/source/LoadedObjectConfigProperty.java

```diff
@@ -0,0 +1,94 @@
+package io.scalecube.config.source;
+
+import io.scalecube.config.ConfigProperty;
+import java.util.Map;
+import java.util.Optional;
+
+// Helper class
+public final class LoadedObjectConfigProperty<T> implements ConfigProperty {
```

Comment:
Where will we use it? now it's never used.

Reply:
In the Scalecube-Configuration-Service (another PR: https://github.com/scalecube/scalecube-configuration-service/pull/122)

Reply:
Removed (it used to do the same as `String`)